code/_core/mob/living/simple/goblin.dm

```diff
@@ -25,7 +25,7 @@
 	blood_type = /reagent/blood/goblin
 	blood_volume = 300
 
-	move_delay = AI_TICK*2
```

Comment:
This should just be AI_TICK as AI_TICK is the fastest the AI can process, so that should generally be its speed.